rest/blip_api_test.go

```diff
@@ -4494,3 +4494,49 @@ func TestBlipLegacyAttachDocUpdate(t *testing.T) {
 		require.Equal(t, keyValueErr.DocumentID, v2Key)
 	}
 }
+
+func TestUnsubChanges(t *testing.T) {
+	base.SetUpTestLogging(t, base.LevelInfo, base.KeyAll)
+	rt := NewRestTester(t, &RestTesterConfig{guestEnabled: true})
+	defer rt.Close()
+
+	btc, err := NewBlipTesterClientOptsWithRT(t, rt, nil)
+	require.NoError(t, err)
+	defer btc.Close()
+
+	// Sub changes
```

Comment:
I think I mentioned in previous comment about making sure it works - but would be nice to bake this into a test assertion:

Can we run an `UnsubChanges` before the first `SubChanges` to make sure this doesn't panic/cause other issues?

Reply:
I did try this, and it did panic initially but it is fixed now. The problem with having it on a test assertion is that Blip recovers from the panic and just logs it instead, so not sure how I could assert on this unless I assert on the logs containing the panic text.

Will try to come up with something.

Reply:
BLIP should return an error code and a message in the body if the handler paniced